.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "neurorad"]
 	path = event_creation/neurorad
 	url = https://github.com/pennmem/neurorad_pipeline.git
+[submodule "neurorad_pipeline"]
```

Comment:
remove this change/commit from the PR, it's a product of our hacking not actually a necessary change